getting-started-2.0.md

```diff
@@ -0,0 +1,155 @@
+   <br>![](./dummy.png "TODO: Create an index.md")
+1. Add some great Markdown content. Feel free to use [this example](https://raw.githubusercontent.com/adobe/helix-example/master/index.md "example index.md") for starters.
+1. Click the _Commit changes_ button.
+1. Go to `https://<my-repo>-<my_org>.project-helix.page/` to admire your work. Replace `<my-repo>` and `<my-org>` with your actual repository and organization names:
+   <br>![](./dummy.png "TODO: Helix Pages rendering your content")
+1. **Congratulations!** 🎉 You have just published your first page using _Helix Pages_!
+
+> **Hint:** You can add as many Markdown files as you like, and link them in Markdown.
+
+> **Note:** If your GitHub organization contains one or more dashes, add an extra dash between the repository and organization name in the _Helix Pages_ URL. For example, if your organization is called `my-org`, and your repository is `test`, the corresponding _Helix Pages_ URL would be `https://test--my-org.project-helix.page/`.
+
+## Step 2: Customize header and footer
+
+_Helix Pages_ can render a header and footer based on Markdown content in your GitHub repository:
+
+1. Add a file called `header.md` to your GitHub repository.
+   <br>![](./dummy.png "TODO: Create a header.md")
+1. Add a logo, and maybe a list of other links you would like to show in the header navigation. Here's some [inspiration](https://raw.githubusercontent.com/adobe/helix-example/master/header.md "example header.md").
+1. Click the _Commit changes_ button.
+1. If you added a logo in step 2, upload that now.
+   <br>![](./dummy.png "TODO: Upload logo")
+1. Add a file called `footer.md` to your GitHub repository.
+1. Add some content you would like to be shown in the footer. Here's a [proposal](https://raw.githubusercontent.com/adobe/helix-example/master/footer.md "example footer.md").
+1. Click the _Commit changes_ button.
+1. Reload `https://<my-repo>-<my_org>.project-helix.page/` to see header and footer in action.
+   <br>![TODO: Helix Pages rendering your header and footer](./dummy.png)
+
+## Step 3: Add your own HTML files
+
+> You'll need:
+> - basic HTML knowledge
+
+If you have existing HTML files or snippets that you wish to reuse, you can simply add them to your GitHub repository and have _Helix_Pages_ merge and deliver them for you:
+
+1. Add a `static.html` file to your GitHub repository. The file name can be anything, just make sure it has a `.html` extension.
+1. Add your HTML. Maybe [like so](https://raw.githubusercontent.com/adobe/helix-example/master/htdocs/static.html "example HTML file")?
+1. Click _Commit changes_.
+1. Refresh `https://<my-repo>-<my_org>.project-helix.page/static.html` to see the rendered result.
+
+> **Note:** If there is a `.md` and `.html` with the same name, _Helix Pages_ will give precendence to the Markdown file. Rename or remove the Markdown file to have the HTML file rendered instead.
```

Comment:
this is not true. (or will not be true, soon). also, the sentence sounds a bit clumsy / complicated.

Reply:
Oh, the I misunderstood https://github.com/adobe/helix-publish/issues/96 :) will it be the other way round?

Reply:
yes. `.html` takes precedence.

Reply:
Thanks. Changed it.